src/ui/input/password_input.jsx

```diff
@@ -61,7 +61,7 @@ export default class PasswordInput extends React.Component {
       <InputWrap
         after={passwordStrength}
         focused={focused}
-        invalidHint={policy ? undefined : invalidHint}
```

Comment:
My assumption is that this logic was added because we didn't want to reposition the password strength tooltip if there was an invalid hint.